packages/core/src/fields/viewFields/fields/MathVF.ts

```diff
@@ -13,12 +13,15 @@ export class MathVF extends AbstractViewField<unknown> {
 	expression?: EvalFunction;
 	expressionStr?: string;
 	hasError: boolean;
+	math: MathJsInstance;
```

Comment:
Why do you add this `math` member variable couldn't you get the singleton directly on line 54?

Reply:
I thought about this and started this way because I expected to need multiple accesses, but than ended up not. 

I was also unsure how this might change depending on the liftime of the singleton / multiple instances, so I left it being initialized by the constructor. 

But fine to change if the singleton-pattern is staying